client/app/components/groups/DeleteGroupButton.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'antd/lib/button';
+import Modal from 'antd/lib/modal';
+import { toastr } from '@/services/ng';
+
+function deleteGroup(event, group, onGroupDeleted) {
+  // prevent default click action on table rows
+  event.preventDefault();
+  event.stopPropagation();
+
+  Modal.confirm({
+    title: 'Delete Group',
+    content: 'Are you sure you want to delete this group?',
+    okText: 'Yes',
+    okType: 'danger',
+    cancelText: 'No',
```

Comment:
Looks like the ramblings of a mad man 😆
"Delete Group! Are you sure? Yes! Danger! No!"